### PR TITLE
Fix invalid event decoding

### PR DESCRIPTION
**Pull request by sz-piotr**

### CODE REVIEW

File: packages/backend/src/Application.ts [hunk 1]
```diff
@@ -201,8 +201,10 @@ export class Application {
       await databaseService.migrateToLatest()
 
       await apiServer.listen()
-      await syncScheduler.start()
-      await blockDownloader.start()
+      if (config.enableSync) {
```
Comment (sin6pi7) [inline packages/backend/src/Application.ts:204]: What is the reason for introducing this?

Comment (sz-piotr) [reply packages/backend/src/Application.ts:204]: I wanted to connect to production database locally but I also didn't want to modify it

Comment (sz-piotr) [reply packages/backend/src/Application.ts:204]: I can extract this to a separate PR if you want

Comment (sin6pi7) [reply packages/backend/src/Application.ts:204]: No, that's fine to be here - simple enough to understand

**Review by sin6pi7 — APPROVED** [review]

**Pull request merged**